hooks/useConnectedWallet.tsx

```diff
@@ -0,0 +1,12 @@
+import { useWallets } from "@privy-io/react-auth"
+
+const useConnectedWallet = () => {
+  const { wallets } = useWallets()
+  const privyWallet = wallets?.find((wallet) => wallet.walletClientType === "privy")
+  const metamaskWallet = wallets?.find((wallet) => wallet.walletClientType === "metamask")
```

Comment:
what happens if they are using a different external wallet?
- rainbow
- zerion
- etc

will your logic still work? 🤔

Reply:
what is the `walletClientType` of the privy wallet? 💭 

can we switch that `===` => `!==` for privy wallet type?

Reply:
```suggestion
  const externalWallet = wallets?.find((wallet) => wallet.walletClientType === "metamask")
```

Reply:
rename to `external` to prevent confusion. not always metamask, ANY external wallet.